Sources/Get/get().swift

```diff
@@ -18,6 +18,10 @@ import Utility
  - Returns: The modules that this manifest requires building
 */
 public func get(manifest: Manifest, manifestParser: (path: String, url: String) throws -> Manifest) throws -> [Package] {
+    if Git.majorVersionNumber < 2 {
```

Comment:
maybe just adding description would be enough? 
We don's use this constant in other places

```swift
// Git 2.0 or higher is required
if Git.majorVersionNumber < 2 {
  throw Error.ObsoleteGitVersion
}
```
vs
```swift
let kMinGitVersionRequired = 2
if Git.majorVersionNumber < kMinGitVersionRequired {
  throw Error.ObsoleteGitVersion
}
```